exec/java-exec/src/main/java/org/apache/drill/exec/server/options/TypeValidators.java

```diff
@@ -141,25 +141,41 @@ public void validate(final OptionValue v, final OptionManager manager) {
 
   public static class BooleanValidator extends TypeValidator {
     public BooleanValidator(String name, boolean def) {
-      super(name, Kind.BOOLEAN, OptionValue.createBoolean(OptionType.SYSTEM, name, def));
+      this(name, def, false);
```

Comment:
Looks file, but should have someone review this who is more familiar with this part of the code.